src/shop.tsx

```diff
@@ -5,18 +5,18 @@ import { Posts } from './routes/posts';
 import { configureStore } from './store';
 import { Albums } from './routes/albums';
 import { GithubSearchRepositories } from './components/RepoSearchForm';
-import RepositoryCard from './components/RepositoryCard';
+import { RepositoryCardContainer } from './components/RepositoryCardContainer';
 
 const store = configureStore();
-
+// TODO: https://flaviocopes.com/react-router-data-from-route/
 export const Shop: React.FC = () => (
     <Provider store={store}>
         <BrowserRouter>
             <Routes>
                 <Route path="/posts" element={<Posts />} />
                 <Route path="/albums" element={<Albums />} />
                 <Route path="/github" element={<GithubSearchRepositories />} />
-                <Route path="/github/:repoParam" element={<RepositoryCard />} />
+                <Route path="/github/:repoParam" element={<RepositoryCardContainer />} />
```

Comment:
Since `repoParam` is [used as a source of Github user & his/her repository name](https://github.com/Murzzik/app-shop/pull/4/files#diff-6b8ddbdf5080a0e302963158124ea2a944b856b1e5d03afe8f98742ff5482fe1R13), it makes sense to have two separate path parameters:
```jsx
<Route path="/github/:username/:repositoryName" .../>
```